code/modules/surgery/tools.dm

```diff
@@ -261,3 +261,134 @@
 			advanced_surgeries |= OC.advanced_surgeries
 		return TRUE
 	return
+
+/obj/item/scalpel/advanced
+	name = "laser scalpel"
+	desc = "An advanced scalpel which uses laser technology to cut. It's set to scalpel mode."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "scalpel_a"
+	hitsound = 'sound/weapons/blade1.ogg'
+	force = 16
+	toolspeed = 0.4
+	light_color = LIGHT_COLOR_GREEN
+
+/obj/item/scalpel/advanced/Initialize()
+	set_light(1)
+	START_PROCESSING(SSobj, src)
+
+/obj/item/scalpel/advanced/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/scalpel/advanced/attack_self(mob/user)
+	playsound(get_turf(user),'sound/machines/click.ogg',50,1)
+	var/obj/item/circular_saw/advanced/saw = new /obj/item/circular_saw/advanced(drop_location())

```

Comment:
this _is_ what the CE tools do but can't you do something with tool behavior or something

also it looks like this should drop it on the ground

Reply:
Never understood why it can't just be 1 tool that has both tool properties anyways. Dunno if surgicals have been moved over to tool behaviors yet (haven't bothered to read up either ;) )

Reply:
both at once might not work if there are two possible steps that both use one of the tool_behaviors on a tool, e.g. screwdriver to disassemble windows/wrench to secure